DOCSP-47239 More Compass Cleanup (#721)
mongodb/docs-compass: source/collections/time-series-collection.txt

:manual:`Time series collections </core/timeseries-collections/>` efficiently 
store sequences of measurements over a period of time. 

Limitations
-----------

The following restrictions and limitations apply when creating a time series 
collection:

- :guilabel:`Custom collation` is the only :guilabel:`Advanced Collection Option` 

>   that can be used alongside your time series collation.  

Reviewer: ```suggestion
  that is usable with a time series collection.  
```
Was "collation" intended, or should this be "collection" as with the suggestion above?

Author: Looks like this was a typo that we didn't catch when this content was first written! Fixed
